src/node_crypto.cc

```diff
@@ -255,13 +255,52 @@ void ThrowCryptoError(Environment* env,
                       unsigned long err,  // NOLINT(runtime/int)
                       const char* default_message = nullptr) {
   HandleScope scope(env->isolate());
+  Local<String> message;
+
   if (err != 0 || default_message == nullptr) {
     char errmsg[128] = { 0 };
     ERR_error_string_n(err, errmsg, sizeof(errmsg));
-    env->ThrowError(errmsg);
+    message = String::NewFromUtf8(env->isolate(), errmsg,
+                                  v8::NewStringType::kNormal)
+                                    .ToLocalChecked();
   } else {
-    env->ThrowError(default_message);
+    message = String::NewFromUtf8(env->isolate(), default_message,
+                                  v8::NewStringType::kNormal)
+                                    .ToLocalChecked();
+  }
+
+  Local<Value> exception_v = Exception::Error(message);
+  CHECK(!exception_v.IsEmpty());
+  // Add the openSSLErrorStack property to the exception object.
+  Local<Object> exception = exception_v.As<Object>();
+  Local<String> key = String::NewFromUtf8(env->isolate(), "openSSLErrorStack",
+                                          v8::NewStringType::kInternalized)
+                                            .ToLocalChecked();
```

Comment:
I think you can feel free to add this to the list in `env.h`. (Also, using `FIXED_ONE_BYTE_STRING` should be okay here too.)